### PR TITLE
Enabled torch.testing._internal.jit_utils.* typechecking.

### PR DESCRIPTION
**Pull request by kauterry**
Fixes #{issue number}

### CODE REVIEW

File: torch/_C/__init__.pyi.in [hunk 1]
```diff
@@ -81,7 +80,30 @@ class finfo:
     @overload
     def __init__(self) -> None: ...
 
-${dtype_class_hints}
+float32: dtype = ...
```
Comment (walterddr) [inline torch/_C/__init__.pyi.in:83]: this is generated. you dont need to explicitly set these.

File: torch/_C/__init__.pyi.in [hunk 2]
```diff
@@ -445,13 +478,43 @@ class ThroughputBenchmark(object):
     def benchmark(self, config: BenchmarkConfig) -> BenchmarkExecutionStats: ...
 
 # IDK if these are actually exposed here, hope they are
-${namedtuple_defs}
+namedtuple_values_indices = NamedTuple("namedtuple_values_indices", [("values", Tensor), ("indices", Tensor)])
```
Comment (walterddr) [inline torch/_C/__init__.pyi.in:481]: same as here.

File: torch/_C/__init__.pyi.in [hunk 3]
```diff
@@ -445,13 +478,43 @@ class ThroughputBenchmark(object):
...
+namedtuple_solution_QR = NamedTuple("namedtuple_solution_QR", [("solution", Tensor), ("QR", Tensor)])
+namedtuple_Q_R = NamedTuple("namedtuple_Q_R", [("Q", Tensor), ("R", Tensor)])
+namedtuple_sign_logabsdet = NamedTuple("namedtuple_sign_logabsdet", [("sign", Tensor), ("logabsdet", Tensor)])
+namedtuple_solution_LU = NamedTuple("namedtuple_solution_LU", [("solution", Tensor), ("LU", Tensor)])
+namedtuple_U_S_V = NamedTuple("namedtuple_U_S_V", [("U", Tensor), ("S", Tensor), ("V", Tensor)])
+namedtuple_solution_cloned_coefficient = NamedTuple("namedtuple_solution_cloned_coefficient", [("solution", Tensor), ("cloned_coefficient", Tensor)])
 
 # Defined in torch/csrc/generic/Storage.cpp
-${legacy_storage_base_hints}
+class DoubleStorageBase(object): ...
```
Comment (walterddr) [inline torch/_C/__init__.pyi.in:492]: same as here.

File: torch/_C/__init__.pyi.in [hunk 4]
```diff
@@ -445,13 +478,43 @@ class ThroughputBenchmark(object):
...
+class BFloat16StorageBase(object): ...
+class ComplexDoubleStorageBase(object): ...
+class ComplexFloatStorageBase(object): ...
+class QUInt8StorageBase(object): ...
+class QInt8StorageBase(object): ...
+class QInt32StorageBase(object): ...
 
 # TODO: where
-${legacy_class_hints}
+class DoubleTensor(Tensor): ...
```
Comment (walterddr) [inline torch/_C/__init__.pyi.in:509]: same as here.

**Review by walterddr — CHANGES_REQUESTED** [review]
thanks for the change @kauterry. overall the change looks good. 

the only thing is that there are some fields in pyi.ini file that you should modify the generator code instead of directly listing out the literals. please comment if you have any trouble finding where those generator codes are.

File: torch/_C/__init__.pyi.in (hunk 1 above)
Comment (kauterry) [reply torch/_C/__init__.pyi.in]: How about now?

**Pull request closed without merging**